gc/base/MemoryPoolAddressOrderedList.hpp

```diff
@@ -60,6 +62,12 @@ class MM_MemoryPoolAddressOrderedList : public MM_MemoryPoolAddressOrderedListBa
 	
 	MM_LargeObjectAllocateStats *_largeObjectCollectorAllocateStats;  /**< Same as _largeObjectAllocateStats except specifically for collector allocates */
 
+	/* for Balanced GC only */
```

Comment:
I'd remove this comment. In fact, we can consider (not in this PR) to replace base performSamplingCalculations with Balanced performSamplingCalculations, so we maintain these count always. Should not be much overhead, since we are sampling. In theory, it could be used in estimates of Live set in ConcurrentGC, although not sure if it would be easy to use and much useful.